README.md

```diff
@@ -13,13 +13,13 @@ Contributing and Communications
 * The [Issues and Milestones](https://github.com/meteor-gazelle/meteor-gazelle/issues) contains items that need to be worked on. Check there for a general overview of what needs to be worked on.
 * IRC
   * &#35;gazelle on irc.what-network.net is the official meteor-gazelle development channel.
-  * If you are looking for something to work on, ping a Core team member (operators in #gazelle).
+  * If you are looking for something to work on, ping the core team using !core <message>, or create a github issue labeled appropriately.
```

Comment:
It's still interpreted as "if you want something to work on then you need to create a github issue"